### PR TITLE
[markdown/en] Made the rendered text more readable.

### PR DESCRIPTION
**Pull request by JacobCWard**
I formatted the document so that the rendered page is more easily
readable. (plus it also serves as even more of an example of how to use
Markdown.) #meta

### CODE REVIEW

**Comment by Spurlow** [comment]
Thanks!

File: markdown.html.markdown [hunk 1]
```diff
@@ -2,45 +2,60 @@
...
 ---
 
-Markdown was created by John Gruber in 2004. It's meant to be an easy to read and write syntax which converts easily to HTML (and now many other formats as well).
-
-Give me as much feedback as you want! / Feel free to fork and pull request!
 
+Markdown was created by John Gruber in 2004. It's meant to be an easy to read and write syntax which converts easily to HTML (and now many other formats as well).
 
-```markdown
-<!-- Markdown is a superset of HTML, so any HTML file is valid Markdown. This
```
Comment (Spurlow) [inline markdown.html.markdown]: This was meant to demonstrate that you can put HTML comments in Markdown.

Comment (JacobCWard) [inline markdown.html.markdown]: See lines 29-31 of the most recent version.

**Comment by vendethiel** [comment]
Thanks! :+1:

**Pull request merged**